Real_Masters_all/mdaily.xml

```diff
@@ -37648,7 +37649,8 @@
               <container type="box" label="Box">40</container>
               <unittitle>"Photo Archive <unitdate type="inclusive" normal="1993/1994">1993-1994</unitdate> Copy</unittitle>
               <physdesc>
-                <physfacet>CD R, black and white and color images in various formats, 1994 images unreadable</physfacet>
+                <extent>1 optical disks</extent>
+                <physfacet>CD-R</physfacet>
```

Comment:
Would it be possible to turn the rest of that original physfacet into an <odd> or something? It seems like in this and various other instances in this finding aid, normalizing the extent leads to some information about the contents and readability of the removable media being lost.